test/tsvector_test.exs

```diff
@@ -0,0 +1,61 @@
+defmodule TsvectorTest do
+  use ExUnit.Case, async: true
+  import Postgrex.TestHelper
+  alias Postgrex, as: P
+  require IEx
+
+  setup do
+    opts = [database: "postgrex_test", backoff_type: :stop]
+    {:ok, pid} = P.start_link(opts)
+    {:ok, [pid: pid]}
+  end
+
+  test "encode basic tsvector", context do
+    assert [[[%Postgrex.Lexeme{positions: [], word: "1"}]]] = query("SELECT $1::tsvector", [[%Postgrex.Lexeme{positions: [], word: "1"}]])
```

Comment:
Added 2 tests using cast to text in the query to make sure the encoding is correct.

Thanks!